chromium_src/components/policy/tools/generate_policy_source.py

```diff
@@ -162,6 +166,27 @@ def AddBravePolicies(template_file_contents):
             'desc': ('''This policy allows an admin to specify that Brave '''
                      '''VPN feature will be disabled.'''),
         },
+        {
+            'name': 'BraveAIChatEnabled',
+            'type': 'main',
+            'schema': {
+                'type': 'boolean'
+            },
+            'supported_on': ['chrome.*:120-'],
```

Comment:
it's `121` already.

Reply:
fixed in 6c38cbfe7ab643bb27407a5e4093316a3b760720